.github/workflows/format.yml

```diff
@@ -4,9 +4,11 @@ on:
   push:
     branches:
       - master
+      - devel
```

Comment:
I think this would be a good opportunity to also enable testing the format on all branches.